mirage/utils.ts

```diff
@@ -0,0 +1,5 @@
+import { faker } from 'ember-cli-mirage';
+
+export function randomGravatar() {
```

Comment:
Accept optional size and use that in the url with `&s=100`. Also replace the institution factory usage.